runtime/foucoco/src/lib.rs

```diff
@@ -1452,6 +1460,30 @@ impl currency::Config for Runtime {
 	type AmountCompatibility = primitives::StellarCompatibility;
 }
 
+parameter_types! {
+	pub const FarmingKeeperPalletId: PalletId = PalletId(*b"bf/fmkpr");
+	pub const FarmingRewardIssuerPalletId: PalletId = PalletId(*b"bf/fmrir");
+}
+parameter_types! {
+	pub BifrostTreasuryAccount: AccountId = TreasuryPalletId::get().into_account_truncating();
+}
+
+type FarmingControlOrigin = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 3, 5>,
```

Comment:
This seems worthy of a discussion. @Rust-Ninja I assume you chose this because you think it's a good fit? Or was this defined elsewhere and you adopted it?

Reply:
I prefer if we just set this to root for now.

Reply:
Done.